sodym/named_dim_array_helper.py

```diff
@@ -0,0 +1,55 @@
+from einops import rearrange
+import numpy as np
+
+from .named_dim_arrays import Flow, NamedDimArray
+from .dimensions import Dimension, DimensionSet
+
+
+def ndarray_stack(ndarrays: list[NamedDimArray], dimension: Dimension) -> NamedDimArray:
```

Comment:
the name ndarray is easily confused with numpy ndarray. Applies also to the variable names in the function.